presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -664,6 +665,11 @@ public SystemSessionProperties(
                         OPTIMIZED_REPARTITIONING_ENABLED,
                         "Experimental: Use optimized repartitioning",
                         featuresConfig.isOptimizedRepartitioningEnabled(),
+                        false),
+                booleanProperty(
+                        SIMPLIFY_ARRAY_OPERATIONS,
+                        "Simplify and optimize array operations.",
+                        false,
```

Comment:
Let's also add a corresponding config in `featuresConfig` and use the configured value as default.

Reply:
Though I would argue code change to make it default to true would be easier than rollout an config, it is always safer to have an off switch.

Reply:
Oops that was a mistake. I want to keep it true by default. Fixed it.